src/scripts/admin/blocks/collapse/item/index.js

```diff
@@ -0,0 +1,97 @@
+import { __ } from '@wordpress/i18n';
+import { registerBlockType } from '@wordpress/blocks';
+import { InnerBlocks, RichText } from '@wordpress/block-editor';
+import { attributes } from './attributes';
+
+registerBlockType('bcgov/collapse-item', {

```

Comment:
namespace change?

Reply:
Honestly thought I had updated that. Must have seen a squirrel when doing so.